src/main/types/CourtPageData.ts

```diff
@@ -1,5 +1,13 @@
+import {Court} from './Court';
+
 export interface CourtPageData {
   isSuperAdmin: boolean,
-  court: {},
+  court: Court,
   updated: boolean
 }
+
+export interface SelectItem {
+  value: any,
```

Comment:
Is it possible to set this as a type rather than an `any` ? To make it a bit more strongly typed